packtools/sps/validation/article_toc_sections.py

```diff
@@ -172,32 +177,54 @@ def validade_article_title_is_different_from_section_titles(self, error_level="E
             ]
         """
         obtained_toc_sections = self.article_toc_sections.sections_dict
-        article_title = self.article_titles.article_title_dict
+        article_titles = self.article_titles.article_title_dict
 
         for lang, sections in obtained_toc_sections.items():
-            is_valid = article_title.get(lang) != sections.get('text')
-            article = article_title.get(lang)
-            section = sections.get("text")
-            if sections.get("parent") == "article":
-                validation_title = 'Article or sub-article section title validation'
-            else:
-                validation_title = f'Sub-article (id={sections.get("parent_id")}) section title validation'
-            yield format_response(
-                title=validation_title,
-                parent=sections.get("parent"),
-                parent_id=sections.get("parent_id"),
-                parent_article_type=sections.get("parent_article_type"),
-                parent_lang=sections.get("parent_lang"),
-                item="subj-group",
-                sub_item="subject",
-                is_valid=is_valid,
-                validation_type="match",
-                expected='\'{}\' (article title) different from \'{}\' (section titles)'.format(article, section),
-                obtained='article title: \'{}\', section titles: \'{}\''.format(article, section),
-                advice="Provide different titles for article and section (subj-group[@subj-group-type='heading']/subject)",
-                data=obtained_toc_sections,
-                error_level=error_level,
-            )
+            for section in sections:
+                article_title = article_titles.get(lang)
+                section_title = section.get("section")
+                is_valid = article_title != section_title
+                if section.get("parent") == "article":
+                    validation_title = 'Article or sub-article section title validation'
+                else:
+                    validation_title = f'Sub-article (id={section.get("parent_id")}) section title validation'
```

Comment:
@Rossi-Luciano title deve ser `Document title must not be similar to section title`

Reply:
O conteúdo de title deve ser similar ao nome do método. Em title deve ter que aspecto está sendo avaliado

Reply:
Deve ser igual não importa se está em article ou sub-article. O valor de parent já dá o contexto.

Reply:
OK, @robertatakenaka